src/model/UserModel.php

```diff
@@ -37,21 +38,36 @@ public function getUsers(): array
 
         $req = $db->prepare('SELECT pseudo FROM user');
         $req->execute();
+
         return $req->fetchAll();
     }
 
-    
     /**
      * @return [type]
```

Comment:
Cela ne veut rien dire. Quel type tu renvoies ? ` [type]`, il faut remplacer cette info.
```suggestion
     * @return [type]
```

Reply:
OK